### PR TITLE
[WIP] Further markup corrections to BIPM Brochure

### PR DESCRIPTION
**Pull request by manuel489**
*In relation to #112*

### CODE REVIEW

**Comment by manuel489** [comment]
Hey @Intelligent2013, I took the liberty of adding you as a reviewer of this PR to see if I'm allowed to make some markup corrections to BIPM Brochure without messing around with your workflow.
Can you take a look, please?
Thanks!

File: sources/sections-a1-en/13-13th-cgpm.adoc [hunk 1]
```diff
@@ -30,7 +30,7 @@ ____
 [[cgpm13th1967r3]]
 === SI unit of thermodynamic temperature (kelvin) (CR, 104 and _Metrologia_, 1968, 4, 43)(((International Temperature Scale of 1990 (ITS-90))))(((kelvin (stem:["unitsml(K)"]))))(((thermodynamic temperature)))
 
-NOTE: At its 1980 meeting, the CIPM approved the report of the 7th meeting of the CCU, which requested that the use of the symbols "stem:["unitsml(degK)"]" and "stem:["unitsml(deg)"]" no longer be permitted.
+NOTE: At its 1980 meeting, the CIPM approved the report of the 7th meeting of the CCU, which requested that the use of the symbols "stem:["unitsml(degK)"]" and "stem:["deg"]" no longer be permitted.
```
Comment (ronaldtse) [inline sources/sections-a1-en/13-13th-cgpm.adoc:33]: If `degK` is encoded as a unit, `deg` is certainly a unit too. This change seems wrong.

File: sources/sections-a1-en/18-16th-cgpm.adoc [hunk 2]
```diff
@@ -64,13 +64,13 @@ The 16th Conférence Générale des Poids et Mesures (CGPM),
 
 *recognizing* the general principles adopted for writing the unit symbols in Resolution 7 of the 9th CGPM (1948),
 
-*considering* that the symbol stem:["unitsml(l)"] for the unit litre was adopted by the Comité International des Poids et Mesures (CIPM) in 1879 and confirmed in the same Resolution of 1948,
+*considering* that the symbol stem:[sf "l"] for the unit litre was adopted by the Comité International des Poids et Mesures (CIPM) in 1879 and confirmed in the same Resolution of 1948,
```
Comment (ronaldtse) [inline sources/sections-a1-en/18-16th-cgpm.adoc:67]: Leave them as UnitsML. This is a UnitsML rendering issue.

File: sources/sections-a1-en/18-16th-cgpm.adoc [hunk 3]
```diff
@@ -64,13 +64,13 @@ The 16th Conférence Générale des Poids et Mesures (CGPM),
...
 
-*considering* also that, in order to avoid the risk of confusion between the letter l and the number 1, several countries have adopted the symbol stem:["unitsml(L)"] instead of stem:["unitsml(l)"] for the unit litre,
+*considering* also that, in order to avoid the risk of confusion between the letter l and the number 1, several countries have adopted the symbol stem:[sf "L"] instead of stem:[sf "l"] for the unit litre,
```
Comment (ronaldtse) [inline sources/sections-a1-en/18-16th-cgpm.adoc:69]: Leave them as UnitsML. This is a UnitsML rendering issue.

File: sources/sections-a1-en/18-16th-cgpm.adoc [hunk 4]
```diff
@@ -64,13 +64,13 @@ The 16th Conférence Générale des Poids et Mesures (CGPM),
...
 
 *considering* that the name litre, although not included in the Système International d'Unités, must be admitted for general use with the System,
 
-*decides*, as an exception, to adopt the two symbols stem:["unitsml(l)"] and stem:["unitsml(L)"] as symbols to be used for the unit litre, footnote:[The CIPM, in 1990, considered that it was still too early to choose a single symbol for the litre.]
+*decides*, as an exception, to adopt the two symbols stem:[sf "l"] and stem:[sf "L"] as symbols to be used for the unit litre, footnote:[The CIPM, in 1990, considered that it was still too early to choose a single symbol for the litre.]
```
Comment (ronaldtse) [inline sources/sections-a1-en/18-16th-cgpm.adoc:73]: Leave them as UnitsML. This is a UnitsML rendering issue.

File: sources/sections-en/05-unit-symbols.adoc [hunk 5]
```diff
@@ -95,8 +95,8 @@ This rule means that the symbol stem:["unitsml(degC)"] for the degree Celsius(((
 ====
 stem:[t = 30.2 "unitsml(degC)"] +
 but not +
-stem:[t = 30.2 "unitsml(degC)"] +
-nor stem:[t = 30.2 "unitsml(degC)"]
+stem:[t = 30.2"°C"] +
+nor stem:[t = 30.2"° C"]
```
Comment (ronaldtse) [inline sources/sections-en/05-unit-symbols.adoc:99]: Leave them as UnitsML. This is a UnitsML rendering issue.

**Review by ronaldtse — CHANGES_REQUESTED** [review]

File: sources/sections-en/05-unit-symbols.adoc [hunk 6]
```diff
@@ -183,7 +188,7 @@ Quantities that are ratios of quantities of the same kind (for example length ra
...
 
-The term "((ppm))", meaning stem:[10^(−6)] relative value, or 1 part in stem:[10^6], or parts per million, is also used. This is analogous to the meaning of percent as parts per hundred. The terms "parts per billion" and "parts per trillion" and their respective abbreviations "((ppb))" and "((ppt))", are also used, but their meanings are language dependent. For this reason the abbreviations ppb and ppt should be avoided.
+The term "((ppm))", meaning stem:[10^(−6)] relative value, or 1 part in stem:[10^6], or parts per million, is also used. This is analogous to the meaning of percent as parts per hundred. The terms "parts per billion" and "parts per trillion" and their respective abbreviations "((ppb))" and "((ppt))"&#x200c;, are also used, but their meanings are language dependent. For this reason the abbreviations ppb and ppt should be avoided.
```
Comment (Intelligent2013) [inline sources/sections-en/05-unit-symbols.adoc:191]: I think we don't need to put something as workaround solution for obvious bug.

**Review by Intelligent2013 — COMMENTED** [review]
@manuel489  I'm not very good spec in stem/unitsml markup, so don't wait response from me.

File: sources/sections-a1-en/13-13th-cgpm.adoc (hunk 1 above)
Comment (manuel489) [reply sources/sections-a1-en/13-13th-cgpm.adoc:33]: @ronaldtse I disagree, "deg" is shown as literal text in original document:

issue1

So, I encoded it as literal text.

File: sources/sections-en/05-unit-symbols.adoc (hunk 5 above)
Comment (manuel489) [reply sources/sections-en/05-unit-symbols.adoc:99]: @ronaldtse, I disagree on this part.
The point of this text is to shown wrong ways of expressing the `°C` unit.
So, we need to reproduce them exactly how it's depicted in original version.

Original version:
issue2

From BIPM:
issue2-1

File: sources/sections-a1-en/18-16th-cgpm.adoc (hunk 2 above)
Comment (manuel489) [reply sources/sections-a1-en/18-16th-cgpm.adoc]: Done

File: sources/sections-a1-en/18-16th-cgpm.adoc (hunk 3 above)
Comment (manuel489) [reply sources/sections-a1-en/18-16th-cgpm.adoc]: Done

File: sources/sections-a1-en/18-16th-cgpm.adoc (hunk 4 above)
Comment (manuel489) [reply sources/sections-a1-en/18-16th-cgpm.adoc]: Done.

File: sources/sections-en/05-unit-symbols.adoc (hunk 6 above)
Comment (manuel489) [reply sources/sections-en/05-unit-symbols.adoc]: Removed. Thanks!

File: sources/sections-en/05-unit-symbols.adoc (hunk 5 above)
Comment (ronaldtse) [reply sources/sections-en/05-unit-symbols.adoc:99]: Ah, I got confused. Thanks!

File: sources/sections-a1-en/13-13th-cgpm.adoc (hunk 1 above)
Comment (ronaldtse) [reply sources/sections-a1-en/13-13th-cgpm.adoc:33]: OK

**Review by ronaldtse — APPROVED** [review]

**Comment by ronaldtse** [comment]
Thanks @manuel489 !

**Pull request merged**